### DISCUSSION TITLE
SFTPGo Server Error Starting Service

### DISCUSSION THREAD

**Discussion by nitaylor** [drakkan/sftpgo#1542] [General]
Hi, I am very new to SFTPGo, and I am having some trouble with the service that runs the software. I was able to install it successfully, and if I restart my computer it runs normally (I am able to transfer files, and configure it via the web admin).

The service runs fine at OS start up, but if I stop the service then try to re-start it, I get "error 1067 the service terminated unexpectedly" The issue is that this will be running on a server environment, so I need to be able to restart the running service without restarting the entire server OS.

I get the same issue running the service via command line.

For reference I am running Windows 10, SFTPGo version 2.5.6

Any ideas would be appreciated. Thanks
![image](https://github.com/drakkan/sftpgo/assets/35982697/c7759713-f7fa-4bf0-98e1-2f0b012e4d80)
![image](https://github.com/drakkan/sftpgo/assets/35982697/e7e13ed4-a5d3-4534-a2dc-ea6c0eecda40)

**Comment by drakkan** [comment]
Hello,

please check or share the SFTPGo logs to understand the issue. Thanks

**Comment by nitaylor** [comment]
Thanks for the quick response. Attached is the log file

**Comment by drakkan** [comment]
> Thanks for the quick response. Attached is the log file [sftpgo.log](https://github.com/drakkan/sftpgo/files/14271551/sftpgo.log)

```
 listen tcp4 :8080: bind: Only one usage of each socket address (protocol/network address/port) is normally permitted
```

Try to change port or to understand what is happening on port 8080

**Comment by nitaylor** [comment]
I was able to change the port in the config file and the service is now able to stop/ start as expected. Using netstat, I can't find a recognizable service that is competing for that port, but it does seem that it was the issue. Thanks for the help
